OpenSSL/crypto.py

```diff
@@ -172,15 +176,18 @@ def generate_key(self, type, bits):
         """
         Generate a key of a given type, with a given number of a bits
 
-        :param type: The key type (TYPE_RSA or TYPE_DSA)
-        :param bits: The number of bits
+        :param type: The key type (TYPE_RSA or TYPE_DSA or TYPE_EC)
+        :param bits: The number of bits or EllipticCurve
```

Comment:
If I was specifying a curve, I'm not sure if I would know how I would do it, based on this. If I was using RSA I would specify something like "2048".  If I was specifying a curve, I would  specify what? "secp384r1"?  I'm not sure if I have a better alternative though or a suggestion for where you should take it.